<p>Avoid hard coded URL in code instead read from the data.xml</p>
<h2>Noncompliant Code Example</h2>
<pre>

getDriver().get("https://www.google.com/") //Noncompliant

</pre>
<h2>Compliant Solution</h2>
<pre>
  
  Read the URL from the data.xml
    
</pre>
