<p>Locators based on css should not be tied to page layout. Page layout can be changed frequently.
  Css locators that contains two or more times a "<code> </code>" (space) or a "<code>></code>" defines it to be tied
  to page layout.</p>
<p>Good locators are unique, descriptive and unlikely to change.</p>
<h2>Noncompliant Code Example</h2>
<pre>
  By.cssSelector("#form div input"); // Noncompliant
</pre>
<h2>Compliant Solution</h2>
<pre>
  By.cssSelector("#submitButton");
</pre>
